module-presentation/src/main/kotlin/site/yourevents/invitation/facade/InvitationFacade.kt

```diff
@@ -39,6 +41,20 @@ class InvitationFacade(
         return CreateInvitationResponse.of(invitation, owner, invitationThumbnail, invitationInformation)
     }
 
+    fun deleteInvitation(
+        invitationId: UUID,
+        authDetails: AuthDetails
+    ) {
+        val invitation = invitationUseCase.findById(invitationId)
+            ?: throw InvitationNotFoundException()
+
+        if(invitation.member.getId() != authDetails.uuid){
+            throw UnauthorizedException()
+        }
+
+        invitationUseCase.deleteInvitation(invitationId,true)
```

Comment:
위와 같은 맥락으로, 메서드명 수정에 대해 고려해주시면 감사하겠습니다!! ❤️ 
그리고 주어지는 invitationId를 기반으로 deleted 필드를 true로 바꾸면 되기에, 매개변수로 boolean 값을 전달해줄 필요는 없어 보입니다!!
```suggestion
        invitationUseCase.markInvitationAsDeleted(invitationId)
```

Reply:
넵!! 마찬가지로 수정하였습니다.